### ISSUE TITLE
Make E2E tests run in parallel

### ISSUE THREAD

**Issue by Efefefef** [LiskHQ/lisk-hub#1271]
### Description of the problem

We need to decrease the time needed to run e2e tests in Jenkins

We can make use of the parallelization feature
Documentation:
https://docs.cypress.io/guides/guides/parallelization.html

**Comment by slaweet** [comment]
Cleaning up - Closing issues that were open for a long time without any update. We can re-open if needed.

**Comment by Efefefef** [comment]
I reopen the issue since we can now make use of the parallelization

**Comment by fchavant** [comment]
@Efefefef does this mean we can get rid of this [workaround](https://github.com/LiskHQ/lisk-hub/pull/1941)?

**Comment by Efefefef** [comment]
@fchavant not yet, found out that we have to wait for the decision on oss plan first now. on hold

**Comment by fchavant** [comment]
Is this ever going to happen? @Efefefef @sridharmeganathan
I've been assigned but blocked since May.

**Comment by Efefefef** [comment]
Closing this as request to cypress still pending and looks like never gonna happen.
